src/scatra/4C_scatra_utils.cpp

```diff
@@ -18,6 +18,52 @@
 
 FOUR_C_NAMESPACE_OPEN
 
+namespace
+{
+  void assert_same_nodes(const Core::Conditions::Condition* const condition1,
```

Comment:
@c-p-schmidt I think this function is too special to warrant a place in core, so I moved it to this place. Actually, it isn't related to conditions that much; it mostly checks whether to vectors contain the same elements.